src/Service/Program.cs

```diff
@@ -27,6 +28,13 @@ public class Program
 
         public static void Main(string[] args)
         {
+            // Compare current version of DAB with latest (non-rc) version in NuGet. 
+            VersionChecker.GetVersions(out string? latestVersion, out string? currentVersion);
+            if (!string.IsNullOrEmpty(latestVersion) && latestVersion != currentVersion)
```

Comment:
what if people clone the repo, their currentVersion might be > latestVersion. Saying newer version is available doesn't make sense in that scenario, we should check for latestVersion > currentVersion